### ISSUE TITLE
Strange results

### ISSUE THREAD

**Issue by ShaoTengLiu** [bryandlee/Tune-A-Video#3]
Hi, thanks for this interesting implementation!

I run the given script on a 3090 and get the following results:

Step0:
<img width="868" alt="截屏2023-01-13 下午12 22 09" src="https://user-images.githubusercontent.com/51499992/212236816-06ec9e04-3033-44f1-a6b7-b1b5406449d3.png">

Step200:
<img width="859" alt="截屏2023-01-13 下午12 22 25" src="https://user-images.githubusercontent.com/51499992/212236843-c33755ab-18ec-4232-a335-1c69ac8adb9a.png">

Step500:
<img width="867" alt="截屏2023-01-13 下午12 23 33" src="https://user-images.githubusercontent.com/51499992/212236987-e019b598-8fef-4818-b3c8-9ec4f8ade526.png">

The results look acceptable at step 200. However, it changes to the original video at step 500.
Could you please give me some hints on this problem? Thank you very much!

**Comment by ExponentialML** [comment]
> The results look acceptable at step 200. However, it changes to the original video at step 500. Could you please give me some hints on this problem? Thank you very much!

I'm getting strange results as well. I'm running the exact same settings, but every single image that I generate looks like *exactly* your first one (steps 100-500). It never converges. Did you change anything in the `.yaml` config to get these results?

Also, in this instance it looks like overfitting. You may be able to resolve this by lowering the learning rate.

**Comment by ShaoTengLiu** [comment]
Hi, thanks for your suggestions.

I don't change anything, just using sample.yml. My environment is torch 1.12.1 + cuda 11.3

**Comment by ShaoTengLiu** [comment]
Using prior_preservation can prevent overfitting to some extent. However, the results are still not stable.

I try to set prior_preservation from 0.1 to 1 but cannot get the results shown in the readme.

<img width="580" alt="截屏2023-01-13 下午4 05 38" src="https://user-images.githubusercontent.com/51499992/212269613-16d6b9e8-0383-4fbc-a3d1-380cc0d0d010.png">

**Comment by bryandlee** [comment]
The model does overfit easily. As @ExponentialML mentioned using smaller learning rates helped in my case.
These are the learning progress I got with the sample.config:

step 0
![step_0](https://user-images.githubusercontent.com/26464535/212292488-c18a0f2b-ccd0-468f-ae89-13a0763936cb.jpg)
step 100
![step_100](https://user-images.githubusercontent.com/26464535/212292495-cb5e2156-5791-47c5-8b0a-aa4edddf6ab7.jpg)
step 200
![step_200](https://user-images.githubusercontent.com/26464535/212292500-b877a890-d34a-41f0-b416-b1b8d33a181a.jpg)
step300
![step_300](https://user-images.githubusercontent.com/26464535/212292503-c71628c9-f3ab-486d-8971-56d900e77469.jpg)
step400
![step_400](https://user-images.githubusercontent.com/26464535/212292505-4c8099cc-d72a-4aa9-8f5d-2922f467db08.jpg)
step500
![step_500](https://user-images.githubusercontent.com/26464535/212292507-4fdc718c-a35b-457f-859c-85945f8f37cd.jpg)

**Comment by ShaoTengLiu** [comment]
Thanks for your reply!

I find the cuda version important. Changing to cuda 11.6 solves my problem.

issue closed.
